src/map/trait.h

```diff
@@ -139,8 +139,11 @@ enum TRAITTYPE
     TRAIT_PRIMEVAL_ZEAL       = 120,
     TRAIT_INSPIRATION         = 121,
     TRAIT_SLEIGHT_OF_SWORD    = 122,
-    TRAIT_DAKEN               = 123
-};
+    TRAIT_DAKEN               = 123,
+    TRAIT_SUPERIOR            = 124, 
+    TRAIT_MAGIC_ACC_BONUS     = 125,
+    TRAIT_MAGIC_EVA_BONUS     = 126
+  };
 
 #define MAX_TRAIT_ID    124

```

Comment:
When you modify any traits, spells, etc., there is a constant you need the change too. 
like ben said, "#define MAX_TRAIT_ID    124" ---> 127